### PR TITLE
Mm 47647 team system statistics card

### PR DESCRIPTION
**Pull request by Afsoon**
#### Summary
Add a new card to the system statistics screen.
Add a new warning and error state to the `active users` card that is colored based if it is over 5% or 10%.
Changing the grid from Bootstrap to CSS Grid.

#### Ticket Link

https://mattermost.atlassian.net/browse/MM-47647

#### Related Pull Requests
- Has server changes https://github.com/mattermost/mattermost-server/pull/21667

#### Screenshots

5% State
Team Statistics
image
System Statistics
image

10%
Team Statistics
image
System Statistics
image

Starter License
image

Cloud License
image

#### Release Note

```release-note
Add Paid Users card to system statistics for self hosted.
Show a warning or error in the Total Active Users Card in case of overage.
```

### CODE REVIEW

**Comment by yasserfaraazkhan** [comment]
Hi @Afsoon ,
Screenshot 2022-11-03 at 12 48 25 PM

will this banner be a part of this change?

**Comment by Afsoon** [comment]
> Hi @Afsoon , Screenshot 2022-11-03 at 12 48 25 PM
> 
> will this banner be a part of this change?

No, the banner will be implemented in this ticket https://mattermost.atlassian.net/browse/MM-47646

File: utils/overage_team.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import {OverActiveUserLimits} from './constants';
```
Comment (nickmisasi) [inline utils/overage_team.ts:3]: I cannot for the life of me find where this is coming from. Doesn't seem to be defined in that file? I'm assuming they're decimal numbers representing the % thresholds?

**Review by nickmisasi — APPROVED** [review]
LGTM!

File: utils/overage_team.ts (hunk 1 above)
Comment (emmyni) [reply utils/overage_team.ts:3]: It's available in the [mm-47645-show-license-count-activated-users](https://github.com/mattermost/mattermost-webapp/tree/mm-47645-show-license-count-activated-users) branch. Probably hasn't been merged into the main branch yet.

**Review by emmyni — APPROVED** [review]
Looks good @Afsoon :)

**Review by yasserfaraazkhan — APPROVED** [review]
Thanks you @Afsoon .
tested this locally, the changes are working as expected.

**Comment by Afsoon** [comment]
gentle ping @annelieseklein

**Comment by Afsoon** [comment]
@annelieseklein gentle ping to review the design of this ticket. Here it's the URL to the instance where you can check it, and it has enough users to force the first threshold. 

URL: https://mattermost-server-pr-21667.test.mattermost.cloud/

| Account Type | Username | Password |
|---|---|---|
| Admin | sysadmin | Sys@dmin123 |
| User | user-1 | User-1@123 |

**Comment by annelieseklein** [comment]
Looks great! can we please add 4px in between the warning and the statistics card?

**Comment by Afsoon** [comment]
@annelieseklein Added the space between the warning message and card 
image

**Comment by annelieseklein** [comment]
Awesome, can you also please align it with the title text? Looks great!

**Review by annelieseklein — APPROVED** [review]

**Pull request merged**